Makefile

```diff
@@ -251,17 +251,21 @@ bundle-register:
 		--current --insecure --tre_url "$${TRE_URL:-https://$${TRE_ID}.$${LOCATION}.cloudapp.azure.com}" --verify \
 		--workspace-service-name "$${WORKSPACE_SERVICE_NAME}"
 
-workspace_bundle = $(MAKE) bundle-build bundle-publish bundle-register \
-	DIR="${MAKEFILE_DIR}/templates/workspaces/$(1)" BUNDLE_TYPE=workspace
+workspace_bundle:
```

Comment:
These targets shouldn't be used anywhere apart from `prepare-for-e2e` target below, so it should be okay to change them